src/http/web_socket.cr

```diff
@@ -90,18 +90,26 @@ class HTTP::WebSocket
     end
   end
 
-  def close(message = nil)
+  # Closes the websocket.
+  def close
     return if closed?
     @closed = true
-    @ws.close(message)
+    @ws.close
+  end
+
+  # Closes the websocket with *code* and *reason*.
+  def close(code : Int16, reason : String? = nil)
```

Comment:
Actually, shouldn't `code` be `UInt16` everywhere?

>a 2-byte unsigned integer (in network byte order) representing a status code with value /code/

Reply:
Why split it into 2 methods?

Reply:
@Sija see in the discussion above

Reply:
Can't find it, which discussion?

Reply:
@Sija See:

https://github.com/crystal-lang/crystal/pull/7483#discussion_r260421899

https://github.com/crystal-lang/crystal/pull/7483#discussion_r260759767